dw-snackbar.js

```diff
@@ -268,23 +234,66 @@ export class DwSnackbar extends layoutMixin(LitElement) {
      * by the user. So, the Toast submitted first will be shown on the top in the sequence.
     */
     this.constructor.counter = 0;
+    this.positionVertical = 'bottom';
+
+    window.addEventListener('resize', this._getLayout.bind(this))
```

Comment:
Event should be bound on attached, and should be unbound when detached. Please, fix it.